libmuscle/python/libmuscle/native_instantiator/test/test_process_manager.py

```diff
@@ -72,7 +72,7 @@ def test_exit_code(lpm, tmp_path):
 def test_multiple(lpm, tmp_path):
     for i in range(3):
         lpm.start(
-                f'test_{i}', tmp_path, ['bash', '-c', 'sleep 1'], {},
+                f'test_{i}', tmp_path, ['bash', '-c', '/usr/bin/env sleep 1'], {},
```

Comment:
Can you clarify how you created a situation in which `/usr/bin` wasn't in the path? `env` (and `ls` and `sleep`) are all POSIX, so I think it's fair to assume that they're available. But maybe I'm missing something?